lib/rentlinx/models/property.rb

```diff
@@ -3,6 +3,7 @@
 require 'rentlinx/modules/linkable'
 
 module Rentlinx
+  # An object that represents a Rentlinx property.
```

Comment:
detail, but seems like sometimes you end sentences with a period, and sometimes you don't.